### PR TITLE
Documentation/jcs/update readme

### LINKED ISSUE #30: Update Readme
### Feature Categories:

- [ ] Angular
- [ ] SCSS / Styling
- [ ] Testing
- [ ] UI 
- [ ] BugFix
- [x] Documentation

**Describe the feature/Bug your gonna work on**

Just updated the readme round 1.

**Additional context**
Add any other context or screenshots about the feature.

### PR DESCRIPTION
**Pull request by JoshSevy**
# Turing Index Pull Request Form

## Description
Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.
- Add dependencies for Deployment hopefully
- Update Readme documentation
- Add all teammates to Readme
- Setup readme template 
## Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Testing (add-testing to project)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Other, Please explain: Documentation
Just updated some of the documentation to keep a road map of the project.
## Has This Been Tested?
- [ ] No
- [x] Yes
Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration:
Visual used markdown viewer in VS Code
# Message for reviewer:
You all are great!
# Checklist:
- [x] My code has no unused/commented out code
- [x] My code has no console logs
- [x] I have performed a self-review of my own code
- [x] I have commented on my code, particularly in hard-to-understand areas

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+language: node_js
```
Comment (NicoleLatifi) [inline .travis.yml:1]: Thanks for adding this .yml file for Travis CI.

File: README.md [hunk 2]
```diff
@@ -59,9 +61,11 @@ A list of commonly used resources that I find helpful are listed in the acknowle
 
 ### Built With
 This section should list any major frameworks that you built your project using. Leave any add-ons/plugins for the acknowledgements section. Here are a few examples.
```
Comment (Leasw144) [inline README.md:63]: We could probably take this section out tbh

File: README.md [hunk 3]
```diff
@@ -44,9 +42,13 @@
...
 
-[![Product Name Screen Shot][product-screenshot]](https://example.com)
+![First Draft Map](./src/assets/docImages/roughMap.png)
 
-There are many great README templates available on GitHub, however, I didn't find one that really suit my needs so I created this enhanced one. I want to create a README template so amazing that it'll be the last one you ever need.
+
+![FirstNav](./src/assets/docImages/firstNav.png)
+
+
+Just because we are learning remote, this doesn't mean you cannot enjoy the Turing campus experience. Come roam the digital halls of Turing and allow us to help you find your digital class room. Just because we are learning from home doesn't mean we can't experience basement life at Turing. As our final project we wanted to bring the Turing experience to future cohorts to enjoy, living the basement life.
```
Comment (NicoleLatifi) [inline README.md:51]: Love it!

**Review by Leasw144 — APPROVED** [review]
Looks solid. Great setup and everything. Thanks for getting it together

**Review by NicoleLatifi — APPROVED** [review]

**Comment by JoshSevy** [comment]
Changes made in future branch

**Pull request closed without merging**